File: GLLara/GLLLightViewController.m
//
//  GLLLightViewController.m
//  GLLara
//
//  Created by Torsten Kammer on 25.02.13.
//  Copyright (c) 2013 Torsten Kammer. All rights reserved.
//

#import "GLLLightViewController.h"

@interface GLLLightViewController ()

@end

@implementation GLLLightViewController

- (id)init
{
    self = [super initWithNibName:@"GLLLightView" bundle:nil];
    if (self) {
        // Initialization code here.
    }
    
    return self;
}

- (void)loadView
{
	// Load explicitly with this method, to make sure it goes through DMLocalizedNibBundle.
	[NSBundle loadNibNamed:self.nibName owner:self];
}

- (IBAction)help:(id)sender;
{
	NSString *locBookName = [[NSBundle mainBundle] objectForInfoDictionaryKey: @"CFBundleHelpBookName"];
	[[NSHelpManager sharedHelpManager] openHelpAnchor:@"diffuselight" inBook:locBookName];
}

@end

File: GLLara/GLLLightViewController.swift
//
//  GLLLightViewController.swift
//  GLLara
//
//  Created by Torsten Kammer on 23.07.22.
//  Copyright © 2022 Torsten Kammer. All rights reserved.
//

import Cocoa

@objc class GLLLightViewController: NSViewController {
    
    convenience init() {
        self.init(nibName: "GLLLightView", bundle: nil)
    }
    
    @IBAction override func showContextHelp(_ sender: Any?) {
        let locBookName = Bundle.main.object(forInfoDictionaryKey: "CFBundleHelpBookName") as! NSString?
        NSHelpManager.shared.openHelpAnchor("diffuselight", inBook: locBookName as NSHelpManager.BookName?)
    }
}
